staging/src/k8s.io/apiserver/pkg/util/flowcontrol/formatting.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"fmt"
+
+	fcfmt "k8s.io/apiserver/pkg/util/flowcontrol/format"
+)
+
+var _ fmt.GoStringer = RequestDigest{}
+
+// GoString produces a golang source expression of the value.
+func (rd RequestDigest) GoString() string {
```

Comment:
This looks approximately identical to the default %#v output, except for using "RequestDigest" instead of "flowcontrol.RequestDigest" for the type name; I'm not sure that's especially valuable.

Reply:
No, there is a bigger difference.  Without this, `fmt.Sprintf("%#+v", someRequestDigest)` produces output with hex pointer values instead of the referenced content.